src/parser/paladin/protection/modules/talents/SanctifiedWrathProtJudgement.tsx

```diff
@@ -0,0 +1,96 @@
+        this.addEventListener(Events.energize.by(SELECTED_PLAYER), this.trackedWastedJudgmentHP);
+    }
+
+    trackJudgmentCasts(event: CastEvent) {
+        if (this.selectedCombatant.hasBuff(SPELLS.AVENGING_WRATH.id)) {
+            this.buffedJudgements += 1;
+        }
+    }
+
+    trackedWastedJudgmentHP(event: EnergizeEvent) {
+        const hasAW: boolean = this.selectedCombatant.hasBuff(SPELLS.AVENGING_WRATH.id);
+        
+        const judgementSource: boolean = event.ability.guid === SPELLS.JUDGMENT_HP_ENERGIZE.id;
+        const wastedHolyPower: boolean = event.waste !== null && event.waste !== undefined && event.waste > 0;
+        if (hasAW && judgementSource) {
+            const wasteDueToSanctifiedWrath: number = this.wasteDueToSanctifiedWrath(event);
+            if (wastedHolyPower && wasteDueToSanctifiedWrath !== 0) {
+                this.holyPowerWastes.push(wasteDueToSanctifiedWrath);
+            }
+        }
+    }
+
+    /**
+     * Judgement can grant any of the following values of HP during Avenging Wrath:
+     * 2 - non-crit with no Holy Avenger Buff
+     * 4 - crit with no Holy Avenger Buff
+     * 3 - non-crit with Holy Avenger Buff
+     * 6 - crit with Holy Avenger Buff
+     * 
+     * To consider the wasted holy power generation to be due to a bad Judgement during Avenging Wrath,
+     * the spell must not be a crit with HA up (wasted HP no matter what), the cast must not have been made with >3 HP on
+     * a non-HA cast (crits are random so we let the wasted HP slide), or the cast must not have been made with >2 HP on a 
+     * HA cast.
+     * @param event 
+     * @returns Number of wasted Holy Power due to Sanctified Wrath talent.
+     */
+    wasteDueToSanctifiedWrath(event: EnergizeEvent): number {
+        const hasHA: boolean = this.selectedCombatant.hasBuff(SPELLS.HOLY_AVENGER_TALENT.id);
+        const hpChange: number = event.resourceChange;
+        const preCastHP = this.MAX_HOLY_POWER - (hpChange - event.waste);
+        const wasCrit: boolean = (hpChange === this.CRIT_NO_HA_CHANGE || hpChange === this.CRIT_YES_HA_CHANGE);
+        if ((hasHA && !wasCrit && preCastHP > 2) || (!hasHA && preCastHP > 3)) {
+            return event.waste;
+        } else {
+            return 0;
+        }
+    }
+
+    statistic() {
+        const totalWastedHP = this.holyPowerWastes.reduce((sum, current) => sum + current, 0);
+        const bonusHP = (this.buffedJudgements * 2) - totalWastedHP;
+        return (
+            <Statistic
+                position={STATISTIC_ORDER.DEFAULT}
+                size="flexible"
+                category={STATISTIC_CATEGORY.TALENTS}
+            >
+                <BoringSpellValue
+                    spell={SPELLS.SANCTIFIED_WRATH_PROT_TALENT}
+                    value={`${formatNumber(bonusHP)} Extra Holy Power`}
+                    label={`${this.buffedJudgements * 2} extra Holy Power generated by Sanctified Wrath of which ${totalWastedHP} was wasted.`}
```

Comment:
This should be:

- value: `formatNumber(bonusHP)`
- label: `Extra Holy Power`
- tooltip (on the statistic): something like `<b>X</b> total additional Holy Power generated by Sanctified Wrath.<br /><b>Y</b> additional Holy Power wasted by overcapping.`

Reply:
Updated screenshots:

<img width="404" alt="2020-10-18 15_21_15-Heroic Wrathion - Kill (2_18) by Jinybb in Ny'alotha - WoWAnalyzer" src="https://user-images.githubusercontent.com/6363390/96387357-46588c80-1156-11eb-8763-0d65c1e29df6.png">
<img width="264" alt="2020-10-18 15_26_41-Heroic Wrathion - Kill (2_18) by Jinybb in Ny'alotha - WoWAnalyzer" src="https://user-images.githubusercontent.com/6363390/96387373-612b0100-1156-11eb-9aaf-192bba586d47.png">